src/asm/lexer.c

```diff
@@ -900,11 +906,12 @@ static void shiftChars(uint8_t distance)
 		/* Now, `distance` is how many bytes to move forward **in the file** */
 	}
 
+	lexerState->colNo += distance;
```

Comment:
Does this interact correctly with expansions? Those should not increase the column count.

Reply:
Yes, in test files with `EQUS`, interpolations, macros, or `rept`s, the reported column numbers match the cursor in the actual source text.

Reply:
Given that `colNo` is currently only used in `dbgPrint`, and due to the nature of that, we can't have a test, so I'll take your word for it.